template/build/webpack.prod.conf.js

```diff
@@ -75,43 +61,12 @@ const webpackConfig = merge(baseWebpackConfig, {
         // more options:
         // https://github.com/kangax/html-minifier#options-quick-reference
       },
-      // necessary to consistently work with multiple chunks via CommonsChunkPlugin
+      // necessary to consistently work with multiple chunks
       chunksSortMode: 'dependency'
     }),
     // keep module.id stable when vendor modules does not change
+    new webpack.NamedChunksPlugin(),
```

Comment:
https://webpack.js.org/guides/caching/#module-identifiers

`HashedModuleIdsPlugin` is enough